routing/router_test.go

```diff
@@ -136,6 +136,7 @@ func createTestCtxFromGraphInstanceAssumeValid(t *testing.T,
 		graphInstance.graphBackend, route.Vertex{}, mcConfig,
 	)
 	require.NoError(t, err, "failed to create missioncontrol")
+	drainMCStoreQueueChan(t, mc.store)
```

Comment:
the addition of this miiight be worth its own commit. Not immediately clear why this is required